Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx

```diff
@@ -149,6 +181,8 @@ vtkSlicerMarkupsWidgetRepresentation3D::vtkSlicerMarkupsWidgetRepresentation3D()
 
   this->AccuratePicker = vtkSmartPointer<vtkCellPicker>::New();
   this->AccuratePicker->SetTolerance(.005);
+
+  this->OccludedRelativeOffset = -25000; /// Default occluded relative offset
```

Comment:
Can you describe in the commend why the value is -25000 (why not less, why not more)?